src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/ACTAccessRequirementWidget.java

```diff
@@ -104,7 +104,7 @@ public void onSuccess(String rootWikiId) {
 		subjectsWidget.configure(ar.getSubjectIds());
 		manageAccessButton.configure(ar);
 		convertACTAccessRequirementButton.configure(ar, refreshCallback);
-		view.setAccessRequirementDescription(ar.getDescription());	
```

Comment:
Searched for all references in the codebase to AccessRequirement.getDescription() and setDescription(), and changed to getName() and setName().